src/project/internal/audacityproject.cpp

```diff
@@ -215,7 +229,7 @@ Ret Audacity4Project::canSave() const
 
     // Ret ret = m_engravingProject->checkCorrupted();
     // if (!ret) {
-    //     Err errorCode = m_engravingProject->isCorruptedUponLoading() ? Err::CorruptionUponOpenningError : Err::CorruptionError;
+    //     Err errorCode = m_engravingProject->isCorruptedUponLoading() ? Err::CorruptionUponOpeningError : Err::CorruptionError;
```

Comment:
I believe this could be safely removed?

Reply:
> should I clean them all?
depending on are they relevant to AU4 or not. Verifying this could be a task in itself. Let's keep it and avoid polluting this PR further.